routers/api/packages/api.go

```diff
@@ -215,6 +215,10 @@ func Routes(ctx gocontext.Context) *web.Route {
 					r.Get("", npm.DownloadPackageFile)
 					r.Delete("/-rev/{revision}", reqPackageAccess(perm.AccessModeWrite), npm.DeletePackageVersion)
 				})
+				r.Group("/-/{filename}", func() {
+					r.Get("", npm.DownloadPackageFile)
+					r.Delete("/-rev/{revision}", reqPackageAccess(perm.AccessModeWrite), npm.DeletePackageVersion)
+				})
```

Comment:
```suggestion
				r.Get("/-/{filename}", npm.DownloadPackageFileByName)
```
The revision endpoints should not be here and they don't work with the changed url.